lib/ansible/plugins/doc_fragments/hwc.py

```diff
@@ -41,10 +41,6 @@ class ModuleDocFragment(object):
         description:
             - The region to which the project belongs.
         type: str
-    id:
-        description:
-            - The id of resource to be managed.
-        type: str
```

Comment:
As wrote above, this needs to be moved to the existing modules using the `hwc` doc fragments.

Reply:
Added id as the input paramters again